press/press/doctype/account_request/account_request.py

```diff
@@ -146,6 +146,8 @@ def too_many_requests_with_field(self, field_name, limits):
 
 	def reset_otp(self):
 		self.otp = generate_otp()
+		if frappe.conf.developer_mode:
+			self.otp = 111111
```

Comment:
best to remove these lines

Reply:
security issue?

Reply:
added an extra check (`frappe.local.dev_server`)